tests/php/src/REST/AdminDashboardControllerTest.php

```
@@ -0,0 +1,481 @@
+        $this->assertTrue( $controller->check_permission( $request ) );
+    }
+
+    /**
+     * Test get_to_do endpoint
+     * @test
+     * @return void
+     */
+    public function test_get_to_do() {
+        wp_set_current_user( $this->admin_id );
+
+        $request = new WP_REST_Request( 'GET', "/{$this->namespace}/{$this->rest_base}/todo" );
```

PHPMD (2.15.0)

Avoid unused local variables such as '$request'. (Unused Code Rules)

(UnusedLocalVariable)